src/datachain/lib/signal_schema.py

```diff
@@ -281,8 +298,22 @@ def clone_without_file_signals(self) -> "SignalSchema":
                 del schema[signal]
         return SignalSchema(schema)
 
+    def has_object(self, name: str) -> bool:
```

Comment:
Renaming is needed. Please avoid using abstract terms like `object`.
`contains()` is enough, `has_signal()` is ok, ...

Reply:
Removed method as it was not needed after refactoring